recipes/openimageio/all/conanfile.py

```diff
@@ -0,0 +1,147 @@
+from conans import ConanFile, CMake, tools
+import os, shutil
+
+class OpenImageIOConan(ConanFile):
+    name = "openimageio"
+    description = "OpenImageIO is a library for reading and writing images, and a bunch of related classes, utilities, and applications." \
+                  "There is a particular emphasis on formats and functionality used in professional, large-scale animation and visual effects work for film."
+    topics = ("conan", "vfx", "image", "picture")
+    license = "BSD-3-Clause"
+    homepage = "http://www.openimageio.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    short_paths = True
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_dicom": [True, False],
+        "with_raw": [True, False]
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_dicom": False, # Heavy dependency, disabled by default
+        "with_raw": False # libraw is available under CDDL-1.0 or LGPL-2.1, for this reason it is disabled by default
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+
+        # CMake options
+        self._cmake.definitions["OIIO_BUILD_TOOLS"] = True
+        self._cmake.definitions["OIIO_BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_DOCS"] = False
+        self._cmake.definitions["INSTALL_DOCS"] = False
+        self._cmake.definitions["INSTALL_FONTS"] = False
+        self._cmake.definitions["INSTALL_CMAKE_HELPER"] = False
+        self._cmake.definitions["EMBEDPLUGINS"] = True
+        self._cmake.definitions["USE_PYTHON"] = False
+        self._cmake.definitions["USE_EXTERNAL_PUGIXML"] = True
+
+        # Set "USE_<PKG>" variables for disabled plugins to False.
+        # If these variables are not set, the package will be built
+        # when required library is found, even if it is not provided
+        # by Conan.
+        self._cmake.definitions["USE_JPEGTURBO"] = False
+        self._cmake.definitions["USE_JPEG"] = True
+        self._cmake.definitions["USE_HDF5"] = False
```

Comment:
I believe this is available

Reply:
HDF5 is needed by Field3D plugin, which also depends on field3d SDK, which is not available.

As for JpegTurbo - the recipe uses libjpeg instead (we can't use both, and some other packages already use libjpeg).

Reply:
Possible solution:

https://github.com/conan-io/conan-center-index/blob/83709600dc900be39d0dcdbf4d866b30649d9d37/recipes/gdal/all/conanfile.py#L34

Could be a separate PR

Reply:
I just saw your commit https://github.com/conan-io/conan-center-index/pull/2017/commits/6771d325fd9890fa60ea2a7c45738ef881726e1b

Now I understand better!